test/modules/environment.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+process.env.TEST = 'test';
```

Comment:
Was this necessary for you?

Reply:
@yuya-oc this the packages context for build and test seems to be different and needed a different `deepmerge` require:
`
let deepmerge = require('deepmerge').default;
if (process.env.TEST) {
  deepmerge = require('deepmerge'); // eslint-disable-line
}
`
https://github.com/mattermost/desktop/pull/586/files#diff-7cdc7778ed182028c72b7c53ac2af20bR4

Reply:
@dmeza Got it. Assuming that, this sentence is not needed. When `src/common/settings.js` requires deepmerge, it correctly uses `src/node_modules/deepmerge`. Probably you met the error of that `test/specs/settings_test.js` requires deepmerge at outside of `src/`.